README.md

```diff
@@ -54,3 +54,9 @@ Value of the *Server Management Subchain ID* can be found by looking at the iden
 
 You are strongly encouraged to verify the content of the scripts. You can then copy those scripts to a machine connected to the Internet and execute them (they only require curl command to be installed). They do not contain any reference to your secret SK1 key.
 
+## Run in Docker container
+```
+sudo docker build -t factom-identity-cli https://github.com/PaulBernier/factom-identity-cli
```

Comment:
It doesn't work with `https://` it should be just github.com/PaulBernier/factom-identity-cli
Also I think a properly set up Docker shouldn't require sudo in front of the commands, could you remove them?